base/applications/mspaint/main.cpp

```diff
@@ -322,7 +322,7 @@ _tWinMain (HINSTANCE hThisInstance, HINSTANCE hPrevInstance, LPTSTR lpszArgument
     sizeboxCenterBottom.Create(scrlClientWindow.m_hWnd, sizeboxPos, NULL, WS_CHILD | WS_VISIBLE);
     sizeboxRightBottom.Create(scrlClientWindow.m_hWnd, sizeboxPos, NULL, WS_CHILD | WS_VISIBLE);
     /* placing the size boxes around the image */
-    imageArea.SendMessage(WM_SIZE, 0, 0);
+    imageArea.PostMessage(WM_SIZE, 0, 0);
```

Comment:
What's the reason of this change?

Reply:
Reverted.
https://github.com/reactos/reactos/commit/1c163bf163d5f315e4869cff3286df946fb0a637